pom.xml

```diff
@@ -117,6 +118,18 @@
       <artifactId>jnr-unixsocket-nodep</artifactId>
       <version>0.3.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>annotations</artifactId>
+      <version>3.0.0</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-step-api</artifactId>
+      <version>${workflow-jenkins-plugin.version}</version>
```

Comment:
Isn't <optional>true</optional> missing here?

This is gonna make compulsory the installation of that part of the workflow plugin when using ssh-agent, right? For me, it does not sound totally right. Or I'm missing something?

I read the feature that requested that PR as: "if workflow is present, then contribute a step to it [but if absent, then just stay still]"